src/Makefile

```diff
@@ -1,8 +1,14 @@
 LBT_ROOT := $(dir $(abspath $(dir $(lastword $(MAKEFILE_LIST)))))
 include $(LBT_ROOT)/src/Make.inc
 
+ifneq ($(OS),WINNT)
+maintarget=$(builddir)/libblastrampoline.$(SHLIB_EXT)
+else
+maintarget=$(builddir)/$(LIB_MAJOR_VERSION)
+endif
```

Comment:
Ok, this change makes more sense now.  But maybe move it together with https://github.com/JuliaLinearAlgebra/libblastrampoline/blob/165da4cfc584613cbc56c25a69750b819dc30436/src/Makefile#L33-L39 instead of scattering if-windows logic everywhere? Or rather move these lines up together with the new ones.

Reply:
Actually, `maintarget` could be something like `$(word 1,$(TARGET_LIBRARIES))` everywhere.

Reply:
Sounds okay; but, it is not easy to test it; I will try it in my stand alone repo first. Be a while. Tim S.